WalletWasabi.Fluent/Helpers/FlyoutShowController.cs

```diff
@@ -0,0 +1,69 @@
+using System.ComponentModel;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+public class FlyoutShowController : IDisposable
+{
+	private readonly FlyoutBase _flyout;
+	private readonly Control _parent;
+	private bool _isForcedOpen;
+	private bool _isOpen;
+
+	public FlyoutShowController(Control parent, FlyoutBase flyout)
+	{
+		_flyout = flyout;
+		_parent = parent;
+	}
+
+	public void Dispose()
+	{
+		_flyout.Closing -= RejectClose;
+	}
+
+	public void SetIsOpen(bool value)
+	{
+		if (_isOpen == value)
+		{
+			return;
+		}
+
+		Toggle(value);
+
+		_isOpen = value;
+	}
```

Comment:
What is the purpose of this? I mean either the `_IsOpen` or the `_isForcedOpen` is unnecessary, so also either `SetIsOpen` or the `Toggle` is unnecessary too. 

One of the variables can be removed, and with that, these two methods could be merged/simplified.

Reply:
Pushed

Reply:
Again, you're right. This can be merge so one of the variables is unnecessary :) 
Merging the 2 classes into 1 caused this duplication.
Thanks!